nomics/README.md

```diff
@@ -337,3 +337,26 @@
   "statusCode": 200
 }
 ```
+
+## Marketcap Endpoint
```

Comment:
If we go toward calling this `currencies` or `ticker` we could name the Endpoint as such and then just add a line for
```
Using the following values as `endpoint` will fill in the corresponding field parameter:
|  endpoint   |    field     |
| :---------: | :----------: |
|   `price`   |   `price`    |
| `marketcap` | `market_cap` |

```